common/formats/ini-file.h

```diff
@@ -105,10 +105,18 @@ class INIFile {
 	/** Reset everything stored in this INI file. */
 	void	clear();
 
-	bool	loadFromFile(const String &filename); /*!< Load configuration from a file. */
-	bool	loadFromFileOrDataFork(const String &filename); /*!< Load configuration from a file in MacBinary format. */
-	bool	loadFromSaveFile(const String &filename); /*!< Load configuration from a save file. */
-	bool	loadFromStream(SeekableReadStream &stream); /*!< Load configuration from a @ref SeekableReadStream. */
+	/**
+	 * Load configuration from a file.
+	 *
+	 * @param filename     Name of an INI file to parse
+	 * @param strictParser Do not allow garbage to be present in the file (default true)
+	 *
+	 * @return  True if file was parsed successfully
+	 */
+	bool	loadFromFile(const String &filename, bool strictParser = true); /*!< Load configuration from a file. */
+	bool	loadFromFileOrDataFork(const String &filename, bool strictParser = true); /*!< Load configuration from a file in MacBinary format. */
```

Comment:
Please add the documentation  here too

Reply:
After every loadFromXXX function should add the documentation